.travis.yml

```diff
@@ -6,27 +6,26 @@ services:
 
 env:
   global:
+    - GO111MODULE="on"
     - PROTOC_RELEASE="https://github.com/google/protobuf/releases/download/v3.6.1/protoc-3.6.1-linux-x86_64.zip"
     - PROTOC_TARGET="${HOME}/protoc"
     - PATH="${PROTOC_TARGET}/bin:${PATH}"
     - secure: Tbet2rxD8QgjthAo+bxt41qbF2wUPTx0difGK5p4yQISK/njTuT5cqcxnOa4GIbyKtNtx0EgGnyVcQJiQkmZiF6Sabf0mtqU/CQ4PmVV76e9bHwA/CrTtudibMn16ozxuuxvhNxFOMQEhwcQOkW93M/Q9FZUEw9/CGpRGFfSzuA=
 
 cache:
   - "${HOME}/protoc"
-  - "${HOME}/gopath/src/github.com/pseudomuto/protoc-gen-doc/vendor"
 
 go:
-  - 1.10.x
+  - 1.12.x
   - master
 
 install:
   - if [ ! -d "${PROTOC_TARGET}" ]; then curl -fsSL "$PROTOC_RELEASE" > "${PROTOC_TARGET}.zip"; fi
   - if [ -f "${PROTOC_TARGET}.zip" ]; then unzip "${PROTOC_TARGET}.zip" -d "${PROTOC_TARGET}"; fi
-  - go get -u github.com/golang/protobuf/{proto,protoc-gen-go}
-  - make setup
+  - go get -u github.com/golang/protobuf/{proto,protoc-gen-go} github.com/haya14busa/goverage
```

Comment:
Probably better to just `go get` them (and not use `-u`), then go should (I think) take the exact versions defined in `go.mod`.